signer/src/main.rs

```diff
@@ -41,22 +48,30 @@ struct SignerArgs {
     /// pending migrations to the database on startup.
     #[clap(long)]
     migrate_db: bool,
+
+    #[clap(short = 'o', long = "output-format", default_value = "pretty")]
+    output_format: Option<LogOutputFormat>,
```

Comment:
It feels more natural to make this a config option. Same thing for the migrate db field too, although that one probably shouldn't be an option at all.

Reply:
Hm, I think I disagree, I often like to quickly change what I'm outputting depending on how I'm running something, and this is a lot easier than messing with long env vars... 
If we were writing to file then yeah I would agree as that's just in general more static, but here we're writing to stdout/err and for me feels more natural to be able to steer that in the same place as it's being executed.

Reply:
Okay sounds good. Might be worth opening an issue for though... maybe